### ISSUE TITLE
[doc] The demo of 'tls_auth_handler'  is not work

### ISSUE THREAD

**Issue by karezachen** [prometheus/client_python#874]
## ENV
tls_auth_handler_demo.py --> code form README.md
```python
from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
from prometheus_client.exposition import tls_handler
def my_auth_handler(url, method, timeout, headers, data):
    certfile = 'client-crt.pem'
    keyfile = 'client-key.pem'
    return tls_auth_handler(url, method, timeout, headers, data, certfile, keyfile)
registry = CollectorRegistry()
g = Gauge('job_last_success_unixtime', 'Last time a batch job successfully finished', registry=registry)
g.set_to_current_time()
push_to_gateway('localhost:9091', job='batchA', registry=registry, handler=my_auth_handler)
```

```shell
python tls_auth_handle_demo.py
```

## LOG

```shell
Traceback (most recent call last):
  File "/private/tmp/tls_auth_handler_demo.py", line 2, in <module>
    from prometheus_client.exposition import tls_handler
ImportError: cannot import name 'tls_handler' from 'prometheus_client.exposition' (/private/tmp/venv/lib/python3.9/site-packages/prometheus_client/exposition.py)
```

## SHOULD BE

https://github.com/prometheus/client_python/blob/4ab083e4dbf82e3bbb2898fc48db18998fda4d45/README.md?plain=1#L490

update to
```python
from prometheus_client.exposition import tls_auth_handler
```

**Comment by csmarchbanks** [comment]
Good catch, would you like to open a PR to fix this?

**Comment by csmarchbanks** [comment]
Fixed in #876
